src/frontend/screens/DownloadManager/index.tsx

```diff
@@ -0,0 +1,141 @@
+import './index.css'
+
+import React, { lazy, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { DMQueueElement } from 'common/types'
+import { UpdateComponent } from 'frontend/components/UI'
+import ProgressHeader from './components/ProgressHeader'
+import DownloadManagerHeader from './DownloadManagerHeader'
+import { downloadManagerStore } from 'frontend/helpers/electronStores'
+
+const DownloadManagerItem = lazy(
+  async () =>
+    import('frontend/screens/DownloadManager/components/DownloadManagerItem')
+)
+
+type DMQueue = {
+  elements: DMQueueElement[]
+  finished: DMQueueElement[]
+}
+
+export default function DownloadManager(): JSX.Element | null {
+  const { t } = useTranslation()
+  const [refreshing, setRefreshing] = useState(false)
+  const [plannendElements, setPlannendElements] = useState<DMQueueElement[]>([])
+  const [currentElement, setCurrentElement] = useState<DMQueueElement>()
+  const [finishedElem, setFinishedElem] = useState<DMQueueElement[]>()
+
+  useEffect(() => {
+    setRefreshing(true)
+    window.api.getDMQueueInformation().then(({ elements }: DMQueue) => {
+      setCurrentElement(elements[0])
+      setPlannendElements([...elements.slice(1)])
+      setRefreshing(false)
+    })
+
+    const removeHandleDMQueueInformation = window.api.handleDMQueueInformation(
+      (e: Electron.IpcRendererEvent, elements: DMQueueElement[]) => {
+        if (elements) {
+          setCurrentElement(elements[0])
+          setPlannendElements([...elements.slice(1)])
+        }
+      }
+    )
+
+    return () => {
+      removeHandleDMQueueInformation()
+    }
+  }, [])
+
+  useEffect(() => {
+    window.api.getDMQueueInformation().then(({ finished }: DMQueue) => {
+      setFinishedElem(finished)
+    })
+  }, [plannendElements.length, currentElement?.params.appName])
+
+  if (refreshing) {
+    return <UpdateComponent />
+  }
+
+  const hasItems = currentElement || finishedElem?.length
+
+  const handleClearList = () => {
+    setFinishedElem([])
+    downloadManagerStore.set('finished', [])
+  }
+
+  return (
+    <>
+      <h4 style={{ paddingTop: 'var(--space-md)' }}>
+        {t('download.manager.title', 'Download Manager (Beta)')}
```

Comment:
Not sure if it makes sense translation-wise, but maybe we can split this in 2 strings so when we move this out of beta we don't need to update all the translations?